src/Controller/Admin/IndexController.php

```diff
@@ -224,6 +226,9 @@ protected function buildPimcoreSettings(
                 $systemSettings['general']['valid_languages'],
                 true
             ),
+            'requiredLanguages' => empty($requiredLangauges) === true
+                ? $systemSettings['general']['valid_languages']
+                : $requiredLangauges,
```

Comment:
empty() function is not necessary because variable always exists
```suggestion
            'requiredLanguages' => $requiredLanguages ?: $systemSettings['general']['valid_languages'],
```